### PR TITLE
don't use `extern crate` anymore in examples

### PR DESCRIPTION
**Pull request by thomwiggers**
We're on Rust 2018 now.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -159,13 +159,13 @@ impl Vob<usize> {
     /// Create a Vob from a `u8` slice. The most significant bit of each byte comes first in the
     /// resulting Vob.
     ///
-    /// If you are running nightly, this method will use the new `reverse_bits` intrinsic.
+    /// If you are running rustc 1.37 or later, this method will use the new `reverse_bits`
```
Comment (ltratt) [inline src/lib.rs]: I think we can drop this part of the comment entirely: the performance of the method isn't something we really want to commit to in the comment IMHO.

**Comment by ltratt** [comment]
Somehow this one entirely passed me by -- sorry! One minor comment, then I think we can get this in.

**Comment by ltratt** [comment]
Assuming that new commit does everything you want (it does what I want!), please squash.

**Comment by thomwiggers** [comment]
If you approve, you can just use the "squash and merge" button, right?

**Comment by ltratt** [comment]
Unfortunately doing it via GitHub means that we get mixed-up commit messages :( So it's better if you squash it yourself, sort out the commit message, force push, and then I merge. Sorry for the extra steps :/

**Comment by thomwiggers** [comment]
I don't think that in this case it would've mattered much.

**Comment by ltratt** [comment]
Thanks!

**Pull request merged**